Source/Extensions/Blazorise.Charts.Annotation/wwwroot/chart.annotation.js

```diff
@@ -0,0 +1,36 @@
+import { getChart } from "../Blazorise.Charts/charts.js?v=1.2.2.0";
+import { parseFunction } from "../Blazorise.Charts/utilities.js?v=1.2.2.0";
+
+export function addAnnotation(canvasId, options) {
+    const chart = getChart(canvasId);
+
+    if (chart) {
+        if (options) {
+            if (!chart.options.plugins) {
+                chart.options.plugins = [];
+            }
+
+            chart.options.plugins.annotation = adjustOptions(options);
+        }
+
+        chart.update();
+    }
+}
+
+function adjustOptions(options) {
+    return { annotations: options };
+}
```

Comment:
do we really need this function? I think it's more confusing to read then just assigning the object to annotation?

Reply:
Well, it adjusts the options for the plugin. I guess its fine.

Reply:
I just see a regular object assignment here. The naming of the function leads me to think this actually does something with the options.
Anyway I'm fine as is.

Reply:
In this case, the annotations options are not complicated. But I wanted to make it similar to another extension.